db/schema.rb

```diff
@@ -149,17 +114,5 @@
   end
 
   add_foreign_key "courses", "departments"
-  add_foreign_key "courses_tutoring_sessions", "courses"
-  add_foreign_key "courses_tutoring_sessions", "tutoring_sessions"
-  add_foreign_key "courses_users", "courses"
-  add_foreign_key "courses_users", "users"
-  add_foreign_key "departments_tutoring_sessions", "departments"
-  add_foreign_key "departments_tutoring_sessions", "tutoring_sessions"
-  add_foreign_key "roles_users", "roles"
-  add_foreign_key "roles_users", "users"
-  add_foreign_key "spartan_sessions_users", "spartan_sessions"
-  add_foreign_key "spartan_sessions_users", "users"
   add_foreign_key "tutoring_sessions", "users", column: "tutor_id"
-  add_foreign_key "tutoring_sessions_users", "tutoring_sessions"
-  add_foreign_key "tutoring_sessions_users", "users"
```

Comment:
Missing foreign-key restraints for the proper join tables. I'm not entirely sure why this is happening, the current dev branch has the proper tables. Are you sure your local dev branch is up-to-date?

Reply:
For some reason, migrating is causing the old tables to reappear. I'm still looking into this, as none of the migrates should do this.

Reply:
so strange, but I think it is good now